mgmt/src/models/external/overlay/vpcpeering.rs

```diff
@@ -187,6 +281,144 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_validate_expose() {
```

Comment:
two things

1. why not make these distinct tests?
2. why not use bolero?

Reply:
1. Not sure what you mean, you propose to move each of the different case to a separate function? They felt close enough (all about testing the validation for `VpcExpose`) that I could group them in a single function, but if you think it's better to have them separated into many smaller functions, I guess that's also a possibility.

2. Because I'm not familiar with bolero and haven't looked into it (or even thought about it, for that matter :face_with_peeking_eye:). I can, if you think it's worth it.

Reply:
Oh, I won't reject the PR on this basis.

I just think that bolero would be notably easier and more effective than this path.

Perhaps I should do a tutorial on how to use it.

Reply:
I think you did a presentation on it at some point? I need to find the link again. If I'm getting confused and you haven't, then yes, I'd gladly watch/read it!